apis/sfapi/volume.go

```diff
@@ -45,6 +45,48 @@ func (c *Client) GetVolumeByID(volID int64) (v Volume, err error) {
 	return volumes[0], nil
 }
 
+// GetVolumeByDockerName tbd
+func (c *Client) GetVolumeByDockerName(n string, acctID int64) (v Volume, err error) {
+	vols, err := c.GetVolumesByDockerName(n, acctID)
+	if err == nil && len(vols) == 1 {
+		return vols[0], nil
+	}
+
+	if len(vols) > 1 {
+		err = fmt.Errorf("Found more than one Volume with DockerName: %s for Account: %d", n, acctID)
+	} else if len(vols) < 1 {
+		err = fmt.Errorf("Failed to find any Volumes with DockerName: %s for Account: %d", n, acctID)
+	}
+	return v, err
+}
+
+// GetVolumesByDockerName tbd
+func (c *Client) GetVolumesByDockerName(dockerName string, acctID int64) (v []Volume, err error) {
```

Comment:
I initially wrote this with a wrapper for the singular form incase we wanted to expand or reuse the "GetVolumes".  In hindsight it hasn't proven to really be useful without the cli (note the original SF version had a cli that used this).

I can condense it, not a huge deal either way I don't think.